admin_manual/enterprise/firewall/file_firewall.rst

```diff
@@ -97,28 +111,25 @@ Regular Expression
 * User group
 * Request URL
 
-You can combine multiple rules into one rule. E.g., if a rule applies to both 
-the support and the qa-team you could write your rule like this::
+You can combine multiple rules into one rule, e.g., if a rule applies to both the support and the qa-team you could write your rule like this:
 
- Regular Expression > ^(support|qa-team)$ > is > User group
+.. code-block:: text
 
-No Manual Editing
------------------
+ Regular Expression > ^(support|qa-team)$ > is > User group
 
-We do not recommend modifying the configuration values directly in your
-``config.php``. These use JSON encoding, so the values are difficult to read 
-and a single typo will break all of your rules.
+.. warning:: 
+   We do not recommend modifying the configuration values directly in your
+   ``config.php``. These use JSON encoding, so the values are difficult to read
+   and a single typo will break all of your rules.
 
 Controlling Access to Folders
------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The easiest way to block access to a folder, starting with ownCloud 9.0, is to use a
-system tag. A new rule type was added which allows you to block access to
-files and folders, where at least one of the parents has a given tag. Now you
-just need to add the tag to the folder or file, and then block the tag with the
-File Firewall.
+The easiest way to block access to a folder, starting with ownCloud 9.0, is to use a system tag. 
+A new rule type was added which allows you to block access to files and folders, where at least one of the parents has a given tag. 
 
-This example blocks access to any folder with the tag "Confidential" from outside access.
+Now you just need to add the tag to the folder or file, and then block the tag with the File Firewall.
+This example blocks access to any folder with the tag "Confidential".
```

Comment:
Hmmm - I think that I kept the "from outside access" bit in the updated screen shot.